api/src/main/java/com/alibaba/nacos/api/config/ConfigFactory.java

```diff
@@ -59,4 +59,19 @@ public static ConfigService createConfigService(String serverAddr) throws NacosE
         return createConfigService(properties);
     }
 
+    /**
+     * Destroy Config Instance's Resources.
+     *
+     * @param configService
+     * @throws NacosException
+     */
+    public static void destroyConfigService(ConfigService configService) throws NacosException{
+        try {
+            configService.shutDown();
+            configService = null;
```

Comment:
This line code is no used.

Reply:
The shutdown method aims to destroy the thread pool resources which managed by it.And "configservice == null" aims to destroy the config service object instance exhibitly.

Reply:
But it's a formal parameter, `configService` only has effect in this method. the out quote do not be effected.